docs/registry.md

```diff
@@ -0,0 +1,60 @@
+ # Registry Support for SOCI
+
+SOCI is supported by most registries. The notable exception to this is Duckerhub, which currently does not meet our technical requirements. 
+To check if your registry of choice is supported, see [#list-of-registry-support](List of Registry Support).
+
+For most use-cases, support is the only concern. However, there is a difference in *how* registries support SOCI that could cause surprising edge cases. The rest of this document is a technical dive into how SOCI indices are stored and retrieved from registries and the surprises you might encounter.
+
+## Registry Requirements
+
+In order for a registry to support SOCI it must have the following features:
+
+1) Accept [OCI Image Manifests](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/manifest.md) with [subject fields](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/manifest.md#image-manifest-property-descriptions) and arbitrary config media types.
```

Comment:
Is this part of OCI 1.0 (or prior to 1.0)? If so it might be good to refer a 1.0.0 tag instead of 1.1.0-rc2 which looks like a new feature to be added in OCI 1.1.

Reply:
I'd like to reference the same version of the spec throughout this doc to reduce confusion. In this case, we actually do need the latest doc which defines the subject field.

Reply:
I'm okay with that. then we need to change v1.1.0-rc1 to v1.1.0-rc2 (both are used in the doc now)

Reply:
They're actually 2 difference specs. Distribution spec v1.1.0-rc1 and Image spec v1.1.0-rc2.

Reply:
oh missed the name part 😅